packages/actions/test/e2e/01-setup.test.ts

```diff
@@ -1,58 +1,225 @@
+                userFunctions,
+                bucketName,
+                objectName,
+                localPath,
+                process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB || "128",
+                Number(process.env.CONFIG_PRESIGNED_URL_EXPIRATION_IN_SECONDS) || 3600
+            )
+
+            expect(success).to.be.true
+        })
+
+        it("should create a new ceremony", async () => {
+            // make sure we are logged in as coordinator
+            await signInWithEmailAndPassword(userAuth, coordinatorEmail, coordinatorPwd)
+
+            const ceremony = fakeCeremoniesData.fakeCeremonyScheduledDynamic
+            const ceremonyBucket = getBucketName(ceremony.data.prefix, `-${randomBytes(5).toString("hex")}`)
+            const circuit = fakeCircuitsData.fakeCircuitSmallNoContributors
+            // 1 create a bucket for the ceremony
+            await createS3Bucket(userFunctions, ceremonyBucket)
+
+            // 2. upload zkey
+            const zkeyLocalFilePath = "/tmp/circuit.zkey"
+            fs.writeFileSync(zkeyLocalFilePath, "zkey")
+
+            const zkeyStorageFilePath = getZkeyStorageFilePath(circuit.data.prefix!, "circuit_00001.zkey")
+
+            await multiPartUpload(
+                userFunctions,
+                bucketName,
+                zkeyStorageFilePath,
+                zkeyLocalFilePath,
+                String(process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB),
+                Number(process.env.CONFIG_PRESIGNED_URL_EXPIRATION_IN_SECONDS)
+            )
+
+            // 3. upload pot
+            const potLocalFilePath = "/tmp/circuit.pot"
+            fs.writeFileSync(potLocalFilePath, "pot")
+
+            const potStorageFilePath = getPotStorageFilePath("testpot.pot")
+            await multiPartUpload(
+                userFunctions,
+                bucketName,
+                potStorageFilePath,
+                potLocalFilePath,
+                String(process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB),
+                Number(process.env.CONFIG_PRESIGNED_URL_EXPIRATION_IN_SECONDS)
+            )
+
+            // 4. upload r1cs
+            const r1csLocalFilePath = "/tmp/circuit.r1cs"
+            fs.writeFileSync(r1csLocalFilePath, "r1cs")
+
+            const r1csStorageFilePath = getR1csStorageFilePath(circuit.data.prefix!, "circuit_00001.r1cs")
+            await multiPartUpload(
+                userFunctions,
+                bucketName,
+                r1csStorageFilePath,
+                r1csLocalFilePath,
+                String(process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB),
+                Number(process.env.CONFIG_PRESIGNED_URL_EXPIRATION_IN_SECONDS)
+            )
+
+            // 5. setup ceremony
+            await setupCeremony(userFunctions, ceremony.data, ceremony.data.prefix!, [circuit.data])
+
+            // 6. confirm
+            const ceremonyDoc = await adminFirestore.collection("ceremonies").doc(ceremony.uid).get()
```

Comment:
This is definitely not enough. You should check every field value + circuits subcollections documents + S3 bucket paths + files hashes.